functions/function/twitter-function/src/test/java/org/springframework/cloud/fn/twitter/users/TwitterUsersFunctionTests.java

```diff
@@ -184,8 +183,7 @@ public void testOne() throws IOException {
 		}
 	}
 
-	@SpringBootConfiguration
-	@EnableAutoConfiguration
+	@SpringBootApplication
```

Comment:
I think that pattern was used to avoid classpath scanning which is present on the `@SpringBootApplication`.
Not sure though if there is a hard reason to avoid such a scanning in this test suite...